zcash_client_backend/src/keys.rs

```diff
@@ -32,6 +39,16 @@ pub fn spending_key(seed: &[u8], coin_type: u32, account: u32) -> ExtendedSpendi
     )
 }
 
+pub fn derive_transparent_address_from_secret_key(
```

Comment:
To make everything work, we will probably also need a way to go from Seed -> SecretKey (and represent that key as a WIF string)

Something that would be similar to this code I hacked together in my local Proof-of-concept:

```rust

// modified from: https://github.com/adityapk00/zecwallet-light-cli/blob/master/lib/src/lightwallet.rs
let ext_t_key = ExtendedPrivKey::with_seed(&seed).unwrap();
let address_sk = ext_t_key
    .derive_private_key(KeyIndex::hardened_from_normalize_index(44).unwrap())
    .unwrap()
    .derive_private_key(
        KeyIndex::hardened_from_normalize_index(NETWORK.coin_type()).unwrap(),
    )
    .unwrap()
    .derive_private_key(KeyIndex::hardened_from_normalize_index(account).unwrap())
    .unwrap()
    .derive_private_key(KeyIndex::Normal(0))
    .unwrap()
    .derive_private_key(KeyIndex::Normal(index))
    .unwrap()
    .private_key;

let pk_hex = address_sk.to_string();
let mut res = [0; 32];
from_hex(&pk_hex, &mut res);

let output = env
    .new_string(to_wif(&res, true))
    .expect("Couldn't create Java string for private key!");
Ok(output.into_inner())
```

And the related Wallet Import Format converstion:

```rust

// borrowed from: Wagu @ https://github.com/AleoHQ/wagyu/blob/217c139466a41a3d6e13917621496b7a304bd2dc/zcash/src/private_key.rs
fn to_wif(secret_key: &[u8], compressed: bool) -> String {
    let mut wif = [0u8; 38];
    wif[0] = 0x80;
    wif[1..33].copy_from_slice(&(secret_key));

    if compressed {
        wif[33] = 0x01;
        let sum = &checksum(&wif[0..34])[0..4];
        wif[34..].copy_from_slice(sum);
        wif.to_base58()
    } else {
        let sum = &checksum(&wif[0..33])[0..4];
        wif[33..37].copy_from_slice(sum);
        wif[..37].to_base58()
    }
}
```

Reply:
Resolved in our pairing!